category/vm/compiler/ir/x86/emitter.cpp

```diff
@@ -2586,6 +2586,47 @@ namespace monad::vm::compiler::native
             std::move(shift), std::move(value), live);
     }
 
+    // Discharge
+    void Emitter::clz()
```

Comment:
Ran the fuzzer locally (with the osaka revision) for some time and it didn't find any counter-example.